go/border/error.go

```diff
@@ -141,6 +142,7 @@ func (r *Router) createSCMPErrorReply(rp *rpkt.RtrPkt, ct scmp.ClassType,
 			}
 		} else if rp.DirFrom == rpkt.DirExternal {
 			reply.InfoF()
+			reply.UpFlag()
```

Comment:
Sorry, I don't really understand what this does. Where is RtrPkt.UpFlag() defined as a function? (I'm not familiar with Go.)